src/acl/acl.ts

```diff
@@ -582,6 +582,13 @@ export async function saveAclFor(
     typeof internal_defaultFetchOptions
   > = internal_defaultFetchOptions
 ): Promise<AclDataset & WithResourceInfo> {
+  if (!hasAccessibleAcl(resource)) {
+    throw new Error(
+      `Could not determine the location of the ACL for the Resource at \`${getSourceUrl(
```

Comment:
BTW, I discussed the question of which delimiter to use for variables in messages, and we both agreed on using '[' and ']. This complete message though is a good example of where that works well, 'cos the subsequent uses in this message for delimiting function names (e.g. `hasAccessibleAcl()` and `saveAclFor()`) can continue to use backticks (as they aren't variable placeholders), thereby making it clearer which parts of that message actually are 'variables'. I'll add a backlog ticket to search-and-replace all uses of backticks.

Reply:
OK, sounds good.